validation/data/1bin_lumi.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
As these files are only used once what is the motivation to have them be separate JSON files instead of just dicts in the code in `tests/test_validation.py`?

Reply:
consistency with the other files.

Reply:
>  the other files.

? Sorry, I'm not tracking. Which files?

Reply:
Oh things like `validation/data/2bin_histosys_example2.json`. Nevermind.